lib/chef/knife/bootstrap.rb

```diff
@@ -496,6 +514,20 @@ def incomplete_policyfile_options?
         (!!config[:policy_name] ^ config[:policy_group])
       end
 
+      # True if the bootstrap version is greater than or equal to 15 or latest.
+      def bootstrap_version_gt_15?
+        !!config[:bootstrap_version] &&
+          (config[:bootstrap_version] == "latest" ||
+           config[:bootstrap_version].split(".").first.to_i >= CHEF_15)
+      end
+
+      # Consider only if current chef version less than 15.
+      # True if both bootstrap node version greater than or equal to 15 or pre-release is set.
+      def target_node_gt_15?
```

Comment:
the `target_node...` part here is misleading because we're not talking about the other node, the target, we're talking about this computer, the workstation. That said, in my comment above I think we don't need this method as we could simply do `Chef::VERSION.to_i >= 15`

Reply:
Okay, thanks!